modules/security-monitoring-cluster-logging.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-monitoring.adoc
+
+[id="security-monitoring-cluster-logging_{context}"]
+= Cluster logging
+
+In {product-title}, you deploy, manage, and upgrade cluster logging
```

Comment:
Shall we talk about in the log perspective rather than a cluster logging perspective? 
A draft:
You can view the container logs, Buildconfig and Deploymentconfig in real-time using the command 'oc log'
Users who have access to a project are able to see the logs for that project by default.	
Users with admin roles could access all container logs.
To protect your logs of future audit and analyse，you can collect, manage, and view both system, container, and audit logs using cluster-logging add-on feature. 
you deploy, manage, and upgrade cluster logging through the Elasticsearch Operator and Cluster Logging Operator.
For more detail, refer to XXXX.

About the cluster administrator can configure access
to system logs and logs for individual projects.  That is Future functionality(v4.6).

Reply:
I changed the heading of this section from "Cluster logging" to just "Logging." Then I reworded the section as you suggested. Please recheck the new wording.